### ISSUE TITLE
No documentation found on production

### ISSUE THREAD

**Issue by brcebn** [Apipie/apipie-rails#206]
Hello,

I am not able to access to my documentation on production environment but everything is working on development environment. 

I have created another issue on StackOverFlow : http://stackoverflow.com/questions/21831205/no-documentation-found-apipie

I have fixed a part of the problem when I have added : 

``` ruby
config.reload_controllers = true if (Rails.env.development? || Rails.env.preprod? )
```

(preprod is my personal production server before the real production).

But this method works only once. After all my application is down because Rails cannot load/reload controllers correctly visibly. 

Does anyone have an idea to fix this issue ? 

Thank you in advance.

**Comment by asampatoor** [comment]
@benebrice I am encountering the same issue what you have faced. and I have tried the solution proposed by @iNecas but it still it didn't work for me. Could anyone help me out in this?

**Comment by benebrice** [comment]
@asampatoor What is your environment? Did you check this file: 
[https://github.com/benebrice/test-apipie/blob/master/config/initializers/apipie.rb](https://github.com/benebrice/test-apipie/blob/master/config/initializers/apipie.rb)

**Comment by asampatoor** [comment]
Running `rake apipie:cache` in the Docker File did the trick for me.
